### PR TITLE
Display hierarchical files.

### PR DESCRIPTION
**Pull request by justinlittman**

### CODE REVIEW

File: app/javascript/controllers/treegrid_controller.js [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+export default class extends Controller {
+  static targets = ['table']
+
+  get visibleRows () {
+    return this.tableTarget.querySelectorAll('tbody > tr:not([hidden])')
+  }
+
+  toggle (event) {
+    console.log('toggle')
```
Comment (mjgiarlo) [inline app/javascript/controllers/treegrid_controller.js:11]: cruft?

File: spec/support/capybara.rb [hunk 2]
```diff
@@ -2,7 +2,8 @@
 
 RSpec.configure do |config|
   config.prepend_before(:example, type: :system) do |example|
-    if ENV['CYPERFUL']
+    # If you can't use Cyperful, a headed test can be helpful for authoring system specs.
+    if ENV['CYPERFUL'] || example.metadata[:headed_test]
```
Comment (mjgiarlo) [inline spec/support/capybara.rb:6]: Worth tossing something in the README for this?

**Review by mjgiarlo — APPROVED** [review]
Approve pending consideration of two minor comments

File: spec/support/capybara.rb (hunk 2 above)
Comment (justinlittman) [reply spec/support/capybara.rb:6]: Yes, done.

File: app/javascript/controllers/treegrid_controller.js (hunk 1 above)
Comment (justinlittman) [reply app/javascript/controllers/treegrid_controller.js]: Removed.

**Pull request merged**